libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -78,7 +79,8 @@ proc init*(_: type[GossipSubParams]): GossipSubParams =
       disconnectBadPeers: false,
       enablePX: false,
       bandwidthEstimatebps: 100_000_000, # 100 Mbps or 12.5 MBps
-      iwantTimeout: 3 * GossipSubHeartbeatInterval
+      iwantTimeout: 3 * GossipSubHeartbeatInterval,
+      overheadRateLimitConfOpt: Opt.none(tuple[bytes: int, interval: Duration])
```

Comment:
```suggestion
      overheadRateLimit: Opt.none(tuple[bytes: int, interval: Duration])
```

Reply:
This is different from the actual rate limit, it is a config. Adding the Opt makes using `withValue` cleaner.

Reply:
Everything in that structure is a config, it's even name GossipSubParams
And since when do we put the type of a variable in its name?

Reply:
Done